mage_ai/data_preparation/templates/transformers/transformer_actions/column/average.py

```diff
@@ -0,0 +1,28 @@
+from mage_ai.data_cleaner.transformer_actions.base import BaseAction
+from mage_ai.data_cleaner.transformer_actions.constants import ActionType, Axis
+from mage_ai.data_cleaner.transformer_actions.utils import build_transformer_action
+from pandas import DataFrame
+
+if 'transformer' not in globals():
+    from mage_ai.data_preparation.decorators import transformer
+
+
+@transformer
+def execute_transformer_action(df: DataFrame, *args, **kwargs) -> DataFrame:
+    """
+    Execute Transformer Action: ActionType.AVERAGE
+    """
+    action = build_transformer_action(
+        df,
+        type=ActionType.AVERAGE,
+        action_code='',  # Enter further filtering condition on rows
+        arguments=[],  # Enter the columns to compute aggregate over
+        axis=Axis.COLUMN,
+        options={'groupby_columns': []},  # Enter columns to group by
+        outputs=[
+            {'uuid': 'new_aggregate_column_1', 'column_type': 'category'},
```

Comment:
For average, let's use number_with_decimals as the column type.
Add comment: Number of outputs should match number arguments.

Reply:
Same for other aggregation transformer actions.

Reply:
Output of average aggregation uses `number_with_decimals` as the column type now, and all aggregations now specify that the number of outputs must the number of arguments